src/ui/Icons/svg-icons/icons.jsx

```diff
@@ -52,5 +55,7 @@ export const icons = {
     bell: Bell,
     questionMark: QuestionMark,
     trash: Trash,
+    trello: Trello,
+    wave: Wave,
```

Comment:
remove the wave from here, its not a icon its a image. move this to pages/homepage/assets folder